src/main/java/org/example/gymcrmsystem/parser/JsonStorageParser.java

```diff
@@ -0,0 +1,55 @@
+package org.example.gymcrmsystem.parser;
+
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class JsonStorageParser<K, T extends Identifiable<K>> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(JsonStorageParser.class);
```

Comment:
We can use Lombok to provide us with an instance of logger https://projectlombok.org/features/log